<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CargoWindow</class>
 <widget class="QMainWindow" name="CargoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:
qlineargradient(spread:pad, x1:0.502, y1:0.0611111, x2:0.527155, y2:1, stop:0 rgba(0, 255, 104, 255), stop:0.971014 rgba(123, 255, 165, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="productNameEdit">
       <property name="font">
        <font>
         <family>思源黑体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="amountLineEdit">
       <property name="font">
        <font>
         <family>思源黑体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="priceLineEdit">
       <property name="font">
        <font>
         <family>思源黑体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>登记</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="returnButton">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
